src/client/diagnostics.rs

```diff
@@ -0,0 +1,133 @@
+use bevy::diagnostic::DiagnosticId;
+use bevy::{
+    diagnostic::{Diagnostic, Diagnostics, RegisterDiagnostic},
+    prelude::*,
+    time::common_conditions::on_timer,
+};
+use std::time::Duration;
+
+/// Replication stats during packet processing.
+///
+/// Flushed to Diagnostics system periodically.
+#[derive(Default, Resource, Debug)]
+pub struct ReplicationStats {
```

Comment:
How about this name?
```suggestion
pub struct ClientStats {
```

Reply:
yes, better - we may end up with ServerStats later too. will change